Autopilot/AttitudeManager/Inc/AttitudeDatatypes.hpp

```diff
@@ -12,15 +14,17 @@ typedef struct
 } PID_Output_t;
 
 #ifdef SPIKE
-    #define L_TAIL_OUT_CHANNEL 0 // Spike has ruddervators
-    #define R_TAIL_OUT_CHANNEL 1
+    constexpr static uint8_t L_TAIL_OUT_CHANNEL {0}; // Spike has ruddervators
+    constexpr static uint8_t R_TAIL_OUT_CHANNEL {1};
 #else
-    #define ELEVATOR_OUT_CHANNEL 0
-    #define RUDDER_OUT_CHANNEL 1
+    constexpr static uint8_t ELEVATOR_OUT_CHANNEL {0};
+    constexpr static uint8_t RUDDER_OUT_CHANNEL {1};
 #endif
 
-#define AILERON_OUT_CHANNEL 2
-#define THROTTLE_OUT_CHANNEL 3
+constexpr static uint8_t AILERON_OUT_CHANNEL {2};
+constexpr static uint8_t THROTTLE_OUT_CHANNEL {3};
+
+constexpr static uint8_t NUM_PWM_CHANNELS {8};
```

Comment:
Moving this to a config file would require some changes on the simulink side. I'm going to make an asana to do this later. I want to merge this in asap